doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -524,6 +524,8 @@ Documentation
   * Section describing how to enable Amazon Frustration-Free Setup (FFS) support in :ref:`ug_matter_configuring_device_identification`.
   * Notes to the :ref:`bluetooth_central_dfu_smp` sample document specifying the intended use of the sample.
   * DevAcademy links to the :ref:`index` and :ref:`getting_started` pages.
+  * Additional user guidance to the :ref:`ug_nrf9160_gs` and :ref:`ug_thingy91_gsg` pages and the correspond Developing with pages.
+    This should make it easier for user to know where to go next.
```

Comment:
Not sure if we need the second statement.
```suggestion
  * Additional user guidance to the :ref:`ug_nrf9160_gs` and :ref:`ug_thingy91_gsg` pages and the corresponding  Developing with pages.
```

Reply:
Agreed. Will take out.